### ISSUE TITLE
He-Man, Thundercats, Transformers, ...

### ISSUE THREAD

**Issue by mileszs** [getfretless/appendix_n#22]
I feel like we should combine 80s cartoons into one listing. 

Honestly, I'm probably influenced by a ton of different cartoons much more heavily than I realize. For instance, I'm not sure I even know how much Looney Tunes has influenced me.

**Comment by dstrus** [comment]
All three of those were big for me.

Looney Tunes was HUGE for me. I quote them possibly every day. It was one of those things where I tended to memorize the credits so I could tell a Bob Clampett short from a Friz Freleng from a Chuck Jones from a Bob McKimson from a Tex Avery. As they were aimed at adults, there's a lot more to them than, say, those series that were based on toys. I vote that Looney Tunes gets its own entry if anyone else was into them.

**Comment by unixmonkey** [comment]
Looney tunes were cornerstone in my Saturday mornings until they kind of just vanished from TV, but calling it an 80's cartoon is a bit strange, since they are mostly from 1930-1969, so I vote Looney Tunes gets its own entry.

In a similar vein (and time-warp), I watched a lot of Flintstones, Jetsons, and Tom & Jerry.

I never could stand He-Man or Transformers, but I did watch ThunderCats occassionally.
